### PR TITLE
Add timestamps

### PR DESCRIPTION
**Pull request by martaroncero**

### CODE REVIEW

File: app/views/bookings/_booking_items.html.erb [hunk 1]
```diff
@@ -16,6 +16,11 @@
                 <%= "You declined an offer for " %>
               <% end %>
               <%= link_to booking.user.first_name, product_path(booking.product) %>
+                 <div class="time">
+                 <!-- <h6><%#= distance_of_time_in_words(booking.created_at, Time.now) %></h6> -->
+                <h6> <%= booking.posted_ago %></h6>
+
+                  </div>
```
Comment (RyanofWoods) [inline app/views/bookings/_booking_items.html.erb]:
```suggestion
                 <div class="time">
                   <h6> <%= booking.posted_ago %></h6>
                 </div>
```

File: app/assets/stylesheets/components/_requests_cards.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.time h6{
+color: #A6A6A6;
+font-size: 12px;
+padding-top: 3px;
```
Comment (RyanofWoods) [inline app/assets/stylesheets/components/_requests_cards.scss]:
```suggestion
  color: #A6A6A6;
  font-size: 12px;
  padding-top: 3px;
```

File: app/views/bookings/_booking_items.html.erb [hunk 3]
```diff
@@ -16,6 +16,9 @@
                 <%= "You declined an offer for " %>
               <% end %>
               <%= link_to booking.user.first_name, product_path(booking.product) %>
+                 <div class="time">
+                   <h6> <%= booking.posted_ago %></h6>
+                 </div>
             </div>
```
Comment (RyanofWoods) [inline app/views/bookings/_booking_items.html.erb:22]:
```suggestion
              <%= link_to booking.user.first_name, product_path(booking.product) %>
              <div class="time">
                <h6><%= booking.posted_ago %></h6>
              </div>
            </div>
```

File: app/controllers/bookings_controller.rb [hunk 4]
```diff
@@ -17,6 +17,7 @@ def seeker_index
     @pending_bookings = Booking.where(user_id: @user, status: "Pending")
     @accepted_bookings = Booking.where(user_id: @user, status: "Accepted")
     @declined_bookings = Booking.where(user_id: @user, status: "Declined")
+    @booking_order = Booking.order(created_at: :desc)
```
Comment (RyanofWoods) [inline app/controllers/bookings_controller.rb:20]: I'm not sure if this variable is ever used?

File: app/controllers/charities_controller.rb [hunk 5]
```diff
@@ -7,6 +7,7 @@ def index
   def show
     @user = User.new
     @charity = Charity.find(params[:id])
+    @booking = Booking.find(params[:id])
```
Comment (RyanofWoods) [inline app/controllers/charities_controller.rb:10]: I'm not sure if this variable is ever used?

File: app/controllers/bookings_controller.rb (hunk 4 above)
Comment (martaroncero) [reply app/controllers/bookings_controller.rb:20]: Not used!

File: app/controllers/charities_controller.rb (hunk 5 above)
Comment (martaroncero) [reply app/controllers/charities_controller.rb:10]: Same! Thanks!

**Pull request merged**